sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -4040,7 +4043,8 @@ public void onTimer(
             }
           };
 
-      PCollection<String> output = pipeline.apply(transform).apply(ParDo.of(fn));
+      PCollection<String> output =
+          pipeline.apply(transform).setIsBoundedInternal(IsBounded.UNBOUNDED).apply(ParDo.of(fn));
```

Comment:
Should not be calling `setIsBoundedInternal` here. Is this just to force streaming mode? We need to just create a separate run of ValidatesRunner that forces streaming mode.